solr/docker/build.gradle

```diff
@@ -67,8 +67,9 @@ dependencies {
     builtBy 'assemblePackaging'
   }
 
-  solrTgz project(path: ":solr:packaging", configuration: 'solrTgz')
-  solrTgzSignature project(path: ":solr:packaging", configuration: 'solrTgzSignature')
+  // NOCOMMIT: Fix the reference to solr:distribution
+  solrTgz project(path: ":solr:distribution:", configuration: 'solrTgz')
```

Comment:
Typically the distribution is the end-node. Here it's not - you need to add artifacts to a named configuration on the distribution project, then that configuration will be resolvable and consumable in docker. Grep for "artifacts {" blocks - this essentially links a task (and its outputs) or an explicit path + build task to a named configuration, which can be then consumed elsewhere. Here the solrTgz configuration was previously exposed this way.

Reply:
I pushed a fix, and now I can build the docker image. However, I don't know what the `solrTgzSignature` config does - I assume it should have got the `.asc` signature to validate the file, but I pointed it at the tgz file since I don't know how to  and it still works?

Reply:
Fixed